src/pool/srv_pool.c

```diff
@@ -1112,6 +1144,12 @@ pool_svc_step_up_cb(struct ds_rsvc *rsvc)
 		goto out;
 	event_cb_registered = true;
 
+	rc = ds_pool_iv_prop_update(svc->ps_pool, prop);
```

Comment:
not sure why need read && update the prop here? and how about if it changed between read and update?

Reply:
well the leader needs to actively propagate the pool property during service step up process. what do you mean between read and update?